Gruntfile.js

```diff
@@ -56,7 +57,10 @@ module.exports = function(grunt) {
         }
       },
       e2e: {
-        src: ['tests/e2e/**/*.js'],
+        src: [
+          'tests/e2e/*.spec.js',
+          'tests/e2e/**/*.js',
```

Comment:
Why is this changing? What does `*.spec.js` mean over `**/*.js`? Doesn't the second one encapsulate the first one?